tests/unit/Config.js

```diff
@@ -254,6 +257,106 @@ describe('Config', () => {
         });
     });
 
+    describe('time options', () => {
+        afterEach(() => {
+            // Clean up the environment variables
+            delete process.env.EXPIRE_ONE_DAY_EARLIER;
```

Comment:
`delete` may reset an otherwise useful (for later tests) env variable...
the better way to set env variables in test is to use `setEnv()`, which will store the earlier value and automatically restore or reset at the end of the test.

Reply:
Yes, agree, I updated the PR.